drivers/espi/espi_mchp_xec.c

```diff
@@ -903,10 +913,19 @@ static void espi_oob_down_isr(const struct device *dev)
 		/* Register is write-on-clear, ensure only 1 bit is affected */
 		ESPI_OOB_REGS->RX_STS = MCHP_ESPI_OOB_RX_STS_DONE;
 
-		/* Disable Rx interrupt */
-		ESPI_OOB_REGS->RX_IEN &= ~MCHP_ESPI_OOB_RX_IEN;
+		evt.evt_details = data->oob_sync_pending;
+		if (data->oob_sync_pending) {
+			data->oob_sync_pending = false;
+			k_sem_give(&data->rx_lock);
+		} else {
+			LOG_WRN("%s async OOB", __func__);
+			/* For now discard packet from master-initiated
+			 * OOB transaction and indicate channel is free again
+			 */
+			ESPI_OOB_REGS->RX_CTRL |= MCHP_ESPI_OOB_RX_CTRL_AVAIL;
+		}
 
-		k_sem_give(&data->rx_lock);
+		espi_send_callbacks(&data->callbacks, dev, evt);
```

Comment:
callback shouldnt be only for master initiated OOBs? If slave is waiting for response @ retrive_oob(), then callback not req right?